requirements.in

```diff
@@ -1,5 +1,5 @@
 bump2version
-Django>2.2
+Django>3.2
```

Comment:
Not sure if this requires running `pip-compile` to update `requirements.txt`? There didn't seem to be any diff when I ran that command

Reply:
Yeah, a `pip-compile` should bump the version(s) in the root `requirements.txt` to then get it to 3.2 in this case.